src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_SIZE = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+
+    private Node<K,V>[] table;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+    }
 
     @Override
     public void put(K key, V value) {
-
+        grow();
+        int keyHash = getHash(key);
+        if (table[keyHash % table.length] == null) {
```

Comment:
Let's create a private method for this
```suggestion
        int bucketIndex = getBuketIndex(hash);
        if (table[bucketIndex] == null) {
```